### ISSUE TITLE
cmake_find_package and cmake_paths documentation example of Zlib

### ISSUE THREAD

**Issue by nesc1** [conan-io/docs#944]
Hi first of all i want to congratulate all of of you in conan about the great job that you are doing...

My issue is only a note, because the documentation about [cmake_find_package](https://docs.conan.io/en/latest/integrations/cmake/cmake_find_package_generator.html) and [cmake_paths](https://docs.conan.io/en/latest/integrations/cmake/cmake_paths_generator.html) is not entirely accurate because of the name of the zlib package, the conan name is lowercase and the the script in the CMake installation directory (/Modules) is uppercase (/usr/share/cmake-3.10/Modules/FindZLIB.cmake) (at least on Ubuntu 18) and the example is neither all uppercase neither all lowercase :)

So your example neither work with the Findzlib.cmake generated by conan when using cmake_find_package neither work with system FindZLIB.cmake because of the name differences (linux)

so a more correct example will be using a library/package with the same name.

Hope i can help a little about improving your documentation.

Side note:
in my example i used like this:
```
cmake_minimum_required(VERSION 3.0)
project(helloworld)
set(CMAKE_MODULE_PATH ${CMAKE_BINARY_DIR} ${CMAKE_MODULE_PATH})
add_executable(helloworld hello.cpp)
find_package(ZLIB)
if(ZLIB_FOUND)
    message(STATUS "zlib was found")
    include_directories(${ZLIB_INCLUDE_DIRS})
    target_link_libraries(helloworld ${ZLIB_LIBRARIES})
else()
    message(STATUS "zlib was NOT found!!!!")
endif()
```
and using both generators, but we are using always the FindZLIB.cmake from the system and not the generated one from conan.

**Comment by danimtb** [comment]
Hi @nesc1 and thanks a lot for the report!

yes, you are right. The name in this example should be lowercase in all cases as the name of the package is `zlib`.

Would you like to contribute with a PR to the documentation fixing this?

**Comment by nesc1** [comment]
what is a PR? and if you explain what i need to do, I'll be happy to help. :)

but note that to demonstrate the functionality expected the name of the package and the name of the findXXX.cmake script must be equal... so zlib is a bad example

**Comment by danimtb** [comment]
PR means "Pull Request", that is the way to contribute changes to a github open source repository.

Don't worry, I will come back with the changes and let you know. Thanks! 😄

**Comment by danimtb** [comment]
closed via #945
